<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZarzadzanieTypamiPracownikow</class>
 <widget class="QWidget" name="ZarzadzanieTypamiPracownikow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTableView" name="tablica"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="ID_Typu">
           <property name="text">
            <string>Id_Typu_Pracownika</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ID_Typu_Edit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="Kompetencje">
           <property name="text">
            <string>Kompetencje</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Kompetencje_Edit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="Koszt">
           <property name="text">
            <string>Koszt Godzinny</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Koszt_Edit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Wroc_Button">
         <property name="text">
          <string>Wróć</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Dodaj_Button">
         <property name="text">
          <string>Dodaj Typ Pracownika</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Usun_Button">
         <property name="text">
          <string>Usuń Typ Pracownika</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Edytuj_Button">
         <property name="text">
          <string>Edytuj Dane</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
